system/limits.go

```diff
@@ -67,28 +67,43 @@ func ParseSecLimitsFile() (*SecLimits, error) {
 func ParseSecLimits(input string) *SecLimits {
 	limits := &SecLimits{Entries: make([]*SecLimitsEntry, 0, 0)}
 	leadingComments := make([]string, 0, 0)
-	for _, line := range strings.Split(input, "\n") {
+	splitInput := strings.Split(input, "\n")
+	noOfLines := len(splitInput)
+	for lineNo, line := range splitInput {
 		line = strings.TrimSpace(line)
 		if strings.HasPrefix(line, "#") {
 			// Line is a comment
 			leadingComments = append(leadingComments, line)
-		} else if fields := consecutiveSpaces.Split(line, -1); len(fields) == 4 {
+		} else if fields := consecutiveSpaces.Split(line, -1); len(fields) == 4 || len(fields) == 3 {
```

Comment:
@angelabriel I think a comment explaining why this logic is needed can help debug in the future, in case of need.

Reply:
I've added 2 comments. Hopefully it will be now (and for the  future) a little bit clearer.